examples/nfs/nfs-data/Dockerfile

```diff
@@ -12,8 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jsafrane/nfsexporter
-MAINTAINER Jan Safranek <jsafrane@redhat.com>
-ADD index.html /mnt/data/index.html
+FROM centos
+MAINTAINER Jan Safranek, jsafrane@redhat.com; Huamin Chen, hchen@redhat.com
+RUN yum -y install /usr/bin/ps nfs-utils && yum clean all
+RUN mkdir -p /exports
+ADD run_nfs.sh /usr/local/bin/
+ADD index.html /tmp/index.html
+RUN chmod 644 /tmp/index.html
 
-ENTRYPOINT ["/usr/local/bin/run_nfs", "/mnt/data"]
+# expose mountd 20048/tcp and nfsd 2049/tcp and rpcbind 111/tcp
+EXPOSE 2049/tcp 20048/tcp 111/tcp 111/udp
```

Comment:
Sorry, I missed that both 111/tcp and 111/udp is exposed in the image.

However, it seems that only 111/tcp is defined in rc and svc.
https://github.com/kubernetes/kubernetes/pull/22665/files#diff-2dd0818b520821f3d109443e9bc79424R23
https://github.com/kubernetes/kubernetes/pull/22665/files#diff-224bfcd27f055737f9893725d1b72c7dR12
Is it OK?
ServicePort and ContainerPort regard ports as TCP if protocol is omitted.
http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_serviceport
http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_containerport